lib/cpp/test/qt/TQTcpServerTest.cpp

```diff
@@ -1,18 +1,23 @@
 #define BOOST_TEST_MODULE TQTcpServerTest
 #include <QTest>
-#include <boost/smart_ptr.hpp>
+
+#ifndef Q_MOC_RUN
+  #include <boost/smart_ptr.hpp>
```

Comment:
Sorry, I didn't get what you mean. In my build, moc conflicts with boost headers, that's why I added it. Do you have a different method?